### DISCUSSION TITLE
How to reply with modal to context menu interaction?

### DISCUSSION THREAD

**Discussion by AzpektDev** [Mateo-tem/discord-modals#28] [Q&A] [answered]
As title tells, how I can reply with modal to a context menu interaction. 
When I am trying `await showModal` i am getting this error:
`Error [INTERACTION_ALREADY_REPLIED]: The reply to this interaction has already been sent or deferred.`

**Comment by uSebazz** [comment, marked as answer]
I think you have `interaction.reply({ ... })` and `interaction.deferReply({ ... })`

**Reply by AzpektDev** [reply]
But how i can make it show that modal?

**Reply by PabloRNC** [reply]
If you have thats two types you must delete one and then the error will be fixed

**Reply by AzpektDev** [reply]
I dont have any of that

**Reply by PabloRNC** [reply]
Can u send us the code
